realsense2_camera/launch/includes/nodelet.launch.xml

```diff
@@ -24,6 +24,7 @@
   <arg name="enable_infra"        default="false"/>
   <arg name="enable_infra1"       default="false"/>
   <arg name="enable_infra2"       default="false"/>
+  <arg name="enable_infra_rgb"    default="false"/>
```

Comment:
Why not name the argument the same as the node(let) `enable_infrargb`? Even the readme indicates the same, so having the same variable name in the launch file would avoid confusion!?

Reply:
Got it, addressed.